backend/src/routes/analytics.py

```
@@ -0,0 +1,84 @@
+        }
+
+        return success_response(data=response_data)
+    except Exception as e:
+        logger.error(f"Error fetching dashboard data: {str(e)}")
+        return error_response('Failed to fetch dashboard data', details=e)
+
+@analytics_bp.route('/transactions', methods=['GET'])
+async def get_transactions():
+    """Get recent transactions/trades"""
+    try:
+        limit = int(request.args.get('limit', 50))
```

Ruff (0.15.0)

[error] Local variable `limit` is assigned to but never used

Remove assignment to unused variable `limit`

(F841)